python/lsst/ts/mtaos/mtaos.py

```diff
@@ -1006,6 +1018,24 @@ async def run_closed_loop(self):
                         self.log.info(f"Visit {visit_id} already processed, skipping.")
                         continue
 
+                    filter_label, elevation = await self.model.get_image_info(
+                        visit_id,
+                        self.camera_name,
+                    )
+
+                    rotation_angle = float(
+                        np.mean(np.array(self.image_rotator[image_in_oods.obsid]))
+                    )
+
+                    if (
+                        np.abs(rotation_angle - self.current_rotator_position)
+                        > self.rotation_angle_limit
+                    ) or (
+                        np.abs(elevation - self.current_elevation_position)
+                        > self.elevation_angle_limit
+                    ):
```

Comment:
Please add a log message here to allow us to track when this condition happens.. Something like:

```
self.log.info(f"Current rotator position: {self.current_rotator_position:.1f}, current elevation: {self.current_elevation_position:.1f}. Image to process was taken at {rotation_angle=} and {elevation=}.  Difference above threshold {self.rotation_angle_limit}/{self.elevation_angle_limit}. Skipping.")
```

Reply:
Please, use the local variable instead of the class attribute in the log message